src/apps/bm_devkit/hello_world/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ set(BRISTLEMOUTH_FILES
     ${SRC_DIR}/lib/bm_core/src/platform/bm_lwip.c
     ${CMAKE_BINARY_DIR}/_deps/bm_common-src/src/platform/bm_freertos.c
     ${CMAKE_BINARY_DIR}/_deps/bm_common-src/src/lib_state_machine.c
+    ${CMAKE_BINARY_DIR}/_deps/bm_common-src/src/timer_callback_handler.c
+    ${CMAKE_BINARY_DIR}/_deps/bm_common-src/third_party/crc/crc16.c
+    ${CMAKE_BINARY_DIR}/_deps/bm_common-src/third_party/crc/crc32.c
```

Comment:
Do you think you can add these to the bm,_common directory so it can be built independentely?